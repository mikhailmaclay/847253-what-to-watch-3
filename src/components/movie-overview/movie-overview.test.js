// Libraries
import React from 'react';
import renderer from 'react-test-renderer';
// Components
import MovieOverview from './movie-overview';

describe(`<MovieOverview/> should render correctly`, () => {
  const props = {
    directors: [`Shia LaBeouf`],
    actors: [`Shia LaBeouf`],
    score: 9.4,
    level: `Awesome`,
    reviewsCount: 3,
    description: [`Just do it! Make your dreams come true!`]
  };

  it(`should render correctly`, () => {
    const result = renderer.create(
        <MovieOverview {...props}/>
    ).toJSON();

    expect(result).toMatchSnapshot();
  });
});
