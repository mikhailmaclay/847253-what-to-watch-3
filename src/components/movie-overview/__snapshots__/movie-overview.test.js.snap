// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieOverview/> should render correctly should render correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      9.4
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Awesome
      </span>
      <span
        className="movie-rating__count"
      >
        3 ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      Just do it! Make your dreams come true!
    </p>
    <p
      className="movie-card__director"
    >
      <b>
        Director
        : 
        Shia LaBeouf
      </b>
    </p>
    <p
      className="movie-card__starring"
    >
      <b>
        Starring: 
        Shia LaBeouf
      </b>
    </p>
  </div>,
]
`;
